packages/pancake-sass/src/sass.js

```diff
@@ -45,7 +45,10 @@ export const GetPath = ( module, modules, baseLocation, npmOrg ) => {
 	let location;
 	npmOrgs.forEach( org => {
 		if( baseLocation.includes( org ) ){
-			location = baseLocation.replace( `${ org }/`, '' );
+			location = baseLocation.replace( `${ org }`, '' );
+
+			// Hack to remove trailing / or \
+			location = location.replace( /[\\/]+$/, '' );
```

Comment:
No need to do this with the above suggestion ^

Reply:
or use [`dirname`](https://nodejs.org/api/path.html#path_path_dirname_path) or at least [`normalize`](https://nodejs.org/api/path.html#path_path_normalize_path).
I'm not 100% sure what you're trying to do. Really all you need is to build the path and the `path` package should take care of the posix separator for you.

https://docs.npmjs.com/files/folders.html#node-modules

Reply:
I've updated this PR recently but basically on windows, when trimming the `org` variable the trailing `/` was throwing off the replace. Adding a `path.sep` instead of `/` fixes this